.github/workflows/release.yaml

```diff
@@ -18,14 +20,20 @@ jobs:
         uses: kubernetes-sigs/release-actions/setup-tejolote@dd08496c83441d6477114cc0555b96d404dacff7 # v0.1.2
 
       - name: Run tejolote 
-        env:
-          TAG: ${{ github.ref_name }}
         run: |
-          tejolote attest --artifacts github://kubernetes-sigs/karpenter/"${TAG}" github://kubernetes-sigs/karpenter/"${{ github.run_id }}" --output karpenter.intoto.json --sign
+          tejolote attest --artifacts github://kubernetes-sigs/karpenter/${{ env.TAG }} github://kubernetes-sigs/karpenter/"${{ github.run_id }}" --output karpenter.intoto.json --sign
+
+      - uses: openvex/generate-vex@31b415924ea0d72ed5f2640f1dee59dea6c2770b
```

Comment:
Can you add the version at the end here as a comment for `generate-vex`

Reply:
[openvex/generate-vex](https://github.com/openvex/generate-vex) doesn't have a release yet. I'm not sure what version we need to mention.